pkg/serviceaccount/jwt.go

```diff
@@ -133,6 +135,8 @@ func (j *jwtTokenAuthenticator) AuthenticateToken(token string) (user.Info, bool
 			return key, nil
 		})
 
+		claims, _ := parsedToken.Claims.(jwt.MapClaims)
```

Comment:
This needs to come after the error check. Otherwise we panic if we ever have a token that isn't a jwt.